lms/djangoapps/django_comment_client/utils.py

```diff
@@ -137,6 +138,29 @@ def get_accessible_discussion_modules(course, user, include_all=False):  # pylin
     ]
 
 
+def get_discussion_module(course, user, discussion_id):
```

Comment:
Some other points about `discussion_category_id_access` is that it checks if the cached key isn't set, it has a fallback, it has slightly different has_access code.  I just need help getting cleared up and I'm wondering if we could update docs to help clarify as well.  Thanks.